### PR TITLE
Update ru translation

### PR DESCRIPTION
**Pull request by volodymyrchernyshev**
Changes:
1. Ukrainian language resource is added.
2. Russian resource is updated to the latest guidelines (lowercase/uppercase  hex sequences )
3. Missing properties in Russian resource  are added, 
4. Small grammar fixes for Russian resource  

Issue link: https://github.com/dbeaver/dbeaver/issues/16473

### CODE REVIEW

**Comment by LonwoLonwo** [comment]
Hello @volodymyrchernyshev 

Thanks for your contribution.

Please, take a look at this PR https://github.com/dbeaver/dbeaver/pull/21768

You are doing the same job. Maybe it will be better to translate other classes?

**Comment by LonwoLonwo** [comment]
Additionally:

2023-11-13 17_55_45-Interactive native2ascii — Mozilla Firefox

**Comment by CrazyTapok-bit** [comment]
> Additionally:

I'm sorry, it seems that this has nothing to do with the Ukrainian language

File: plugins/org.jkiss.dbeaver.core/OSGI-INF/l10n/bundle_ru.properties [hunk 1]
```diff
@@ -2,118 +2,168 @@
...
+
+editor.entity.name        = \u0420\u0435\u0434\u0430\u043a\u0442\u043e\u0440 \u043e\u0431\u044a\u0435\u043a\u0442\u043e\u0432
+
+menu.database                                                 = \u0411\u0430\u0437\u0430 \u0434\u0430\u043d\u043d\u044b\u0445
+menu.filter.label                                             = \u0424\u0438\u043b\u044c\u0442\u0440
+menu.navigate                                                 = \u041d\u0430\u0432\u0438\u0433\u0430\u0446\u0438\u044f
+
+toolbar.dbeaver-datasource-selector.label = \u0412\u044b\u0431\u043e\u0440 \u0441\u043e\u0435\u0434\u0438\u043d\u0435\u043d\u0438\u044f
+toolbar.dbeaver-edit.label                = \u0420\u0435\u0434\u0430\u043a\u0442\u0438\u0440\u043e\u0432\u0430\u043d\u0438\u0435
+toolbar.dbeaver-general.label             = \u0418\u043d\u0441\u0442\u0440\u0443\u0435\u043d\u0442\u044b \u0411\u0414
```
Comment (LonwoLonwo) [inline plugins/org.jkiss.dbeaver.core/OSGI-INF/l10n/bundle_ru.properties:68]:
```suggestion
toolbar.dbeaver-general.label             = \u0418\u043d\u0441\u0442\u0440\u0443\u043c\u0435\u043d\u0442\u044b \u0411\u0414
```

File: plugins/org.jkiss.dbeaver.core/OSGI-INF/l10n/bundle_ru.properties [hunk 2]
```diff
@@ -2,118 +2,168 @@
...
+
+confirm.driver_download.title=\u0417\u0430\u0433\u0440\u0443\u0437\u0438\u0442\u044c \u0434\u0440\u0430\u0439\u0432\u0435\u0440 \u0411\u0414
+confirm.driver_download.message=\u0424\u0430\u0439\u043b\u044b \u0434\u0440\u0430\u0439\u0432\u0435\u0440\u0430 \u043e\u0442\u0441\u0443\u0442\u0441\u0442\u0432\u0443\u044e\u0442 \u0434\u043b\u044f \u0411\u0414 "{0}".\n\u0425\u043e\u0442\u0438\u0442\u0435 \u043b\u0438 \u0412\u044b \u0437\u0430\u0433\u0440\u0443\u0437\u0438\u0442\u044c \u044d\u0442\u043e\u0442 \u0434\u0440\u0430\u0439\u0432\u0435\u0440 \u0430\u0432\u0442\u043e\u043c\u0430\u0442\u0438\u0447\u0435\u0441\u043a\u0438?
+
+content-type.org.jkiss.dbeaver.shortcut.name=\u042f\u0440\u043b\u044b\u043a
+command.org.jkiss.dbeaver.ui.eula.showPopup.name=\u041f\u043e\u043a\u0430\u0437\u0430\u0442\u044c \u041f\u043e\u043b\u044c\u0437\u043e\u0432\u0430\u0442\u0435\u043b\u044c\u0441\u043a\u043e\u0435 \u0421\u043e\u0433\u043b\u0430\u0449\u0435\u043d\u0438\u0435
```
Comment (LonwoLonwo) [inline plugins/org.jkiss.dbeaver.core/OSGI-INF/l10n/bundle_ru.properties:121]:
```suggestion
command.org.jkiss.dbeaver.ui.eula.showPopup.name=\u041f\u043e\u043a\u0430\u0437\u0430\u0442\u044c \u043f\u043e\u043b\u044c\u0437\u043e\u0432\u0430\u0442\u0435\u043b\u044c\u0441\u043a\u043e\u0435 \u0441\u043e\u0433\u043b\u0430\u0448\u0435\u043d\u0438\u0435 
```

File: plugins/org.jkiss.dbeaver.core/OSGI-INF/l10n/bundle_ru.properties [hunk 3]
```diff
@@ -2,118 +2,168 @@
...
+confirm.driver_download.message=\u0424\u0430\u0439\u043b\u044b \u0434\u0440\u0430\u0439\u0432\u0435\u0440\u0430 \u043e\u0442\u0441\u0443\u0442\u0441\u0442\u0432\u0443\u044e\u0442 \u0434\u043b\u044f \u0411\u0414 "{0}".\n\u0425\u043e\u0442\u0438\u0442\u0435 \u043b\u0438 \u0412\u044b \u0437\u0430\u0433\u0440\u0443\u0437\u0438\u0442\u044c \u044d\u0442\u043e\u0442 \u0434\u0440\u0430\u0439\u0432\u0435\u0440 \u0430\u0432\u0442\u043e\u043c\u0430\u0442\u0438\u0447\u0435\u0441\u043a\u0438?
+
+content-type.org.jkiss.dbeaver.shortcut.name=\u042f\u0440\u043b\u044b\u043a
+command.org.jkiss.dbeaver.ui.eula.showPopup.name=\u041f\u043e\u043a\u0430\u0437\u0430\u0442\u044c \u041f\u043e\u043b\u044c\u0437\u043e\u0432\u0430\u0442\u0435\u043b\u044c\u0441\u043a\u043e\u0435 \u0421\u043e\u0433\u043b\u0430\u0449\u0435\u043d\u0438\u0435 
+command.org.jkiss.dbeaver.ui.eula.showPopup.description=\u041f\u043e\u043a\u0430\u0437\u0430\u0442\u044c \u0434\u0438\u0430\u043b\u043e\u0433 \u0441 \u043d\u043e\u0440\u043c\u0430\u043c\u0438 \u041f\u043e\u043b\u044c\u0437\u043e\u0432\u0430\u0442\u0435\u043b\u044c\u0441\u043a\u043e\u0433\u043e \u0421\u043e\u0433\u043b\u0430\u0448\u0435\u043d\u0438\u044f
```
Comment (LonwoLonwo) [inline plugins/org.jkiss.dbeaver.core/OSGI-INF/l10n/bundle_ru.properties:122]:
```suggestion
command.org.jkiss.dbeaver.ui.eula.showPopup.description=\u041f\u043e\u043a\u0430\u0437\u0430\u0442\u044c \u0434\u0438\u0430\u043b\u043e\u0433 \u0441 \u0443\u0441\u043b\u043e\u0432\u0438\u044f\u043c\u0438 \u043f\u043e\u043b\u044c\u0437\u043e\u0432\u0430\u0442\u0435\u043b\u044c\u0441\u043a\u043e\u0433\u043e \u0441\u043e\u0433\u043b\u0430\u0448\u0435\u043d\u0438\u044f
```

File: plugins/org.jkiss.dbeaver.core/OSGI-INF/l10n/bundle_ru.properties [hunk 4]
```diff
@@ -2,118 +2,168 @@
...
+view.query.manager.description=\u041f\u0440\u043e\u0441\u043c\u043e\u0442\u0440 \u0438 \u043f\u043e\u0438\u0441\u043a \u0432\u0441\u0435\u0445 \u0437\u0430\u043f\u0440\u043e\u0441\u043e\u0432 \u043f\u0440\u0438\u043b\u043e\u0436\u0435\u043d\u0438\u044f. \u041f\u0440\u043e\u0441\u043c\u043e\u0442\u0440 \u043e\u0448\u0438\u0431\u043e\u043a \u0437\u0430\u043f\u0440\u043e\u0441\u043e\u0432 \u0438 \u0441\u043a\u0440\u044b\u0442\u044b\u0445 \u0437\u0430\u043f\u0440\u043e\u0441\u043e\u0432.
+view.shell.process.description=\u041c\u043e\u043d\u0438\u0442\u043e\u0440\u0438\u043d\u0433 \u0438 \u043a\u043e\u043d\u0442\u0440\u043e\u043b\u044c \u043f\u0440\u043e\u0446\u0435\u0441\u0441\u0430 \u043a\u043e\u043c\u043c\u0430\u043d\u0434\u043d\u043e\u0439 \u0441\u0442\u0440\u043e\u043a\u0438.
```
Comment (LonwoLonwo) [inline plugins/org.jkiss.dbeaver.core/OSGI-INF/l10n/bundle_ru.properties:124]:
```suggestion
view.shell.process.description=\u041c\u043e\u043d\u0438\u0442\u043e\u0440\u0438\u043d\u0433 \u0438 \u043a\u043e\u043d\u0442\u0440\u043e\u043b\u044c \u043f\u0440\u043e\u0446\u0435\u0441\u0441\u0430 \u043a\u043e\u043c\u0430\u043d\u0434\u043d\u043e\u0439 \u0441\u0442\u0440\u043e\u043a\u0438. 
```

File: plugins/org.jkiss.dbeaver.core/OSGI-INF/l10n/bundle_ru.properties [hunk 5]
```diff
@@ -2,118 +2,168 @@
...
+view.query.manager.description=\u041f\u0440\u043e\u0441\u043c\u043e\u0442\u0440 \u0438 \u043f\u043e\u0438\u0441\u043a \u0432\u0441\u0435\u0445 \u0437\u0430\u043f\u0440\u043e\u0441\u043e\u0432 \u043f\u0440\u0438\u043b\u043e\u0436\u0435\u043d\u0438\u044f. \u041f\u0440\u043e\u0441\u043c\u043e\u0442\u0440 \u043e\u0448\u0438\u0431\u043e\u043a \u0437\u0430\u043f\u0440\u043e\u0441\u043e\u0432 \u0438 \u0441\u043a\u0440\u044b\u0442\u044b\u0445 \u0437\u0430\u043f\u0440\u043e\u0441\u043e\u0432.
+view.shell.process.description=\u041c\u043e\u043d\u0438\u0442\u043e\u0440\u0438\u043d\u0433 \u0438 \u043a\u043e\u043d\u0442\u0440\u043e\u043b\u044c \u043f\u0440\u043e\u0446\u0435\u0441\u0441\u0430 \u043a\u043e\u043c\u043c\u0430\u043d\u0434\u043d\u043e\u0439 \u0441\u0442\u0440\u043e\u043a\u0438. 
+view.database.output.title=\u0412\u0438\u0432\u043e\u0434 \u0434\u0430\u043d\u043d\u044b\u0445
```
Comment (LonwoLonwo) [inline plugins/org.jkiss.dbeaver.core/OSGI-INF/l10n/bundle_ru.properties:125]:
```suggestion
view.database.output.title=\u0412\u044b\u0432\u043e\u0434 \u0434\u0430\u043d\u043d\u044b\u0445
```

File: plugins/org.jkiss.dbeaver.core/OSGI-INF/l10n/bundle_ru.properties [hunk 6]
```diff
@@ -2,118 +2,168 @@
...
+command.org.eclipse.ui.navigate.collapseAll.name=\u0421\u0432\u0435\u0440\u043d\u0443\u0442\u044c \u0432\u0441\u0451
+command.org.eclipse.ui.navigate.expandAll.name=\u0420\u0430\u0437\u0432\u0435\u0440\u043d\u0443\u0442\u044c \u0432\u0441\u0451
+command.org.eclipse.ui.views.showView.queryManager=\u041c\u0435\u043d\u0435\u0434\u0436\u0435\u0440 \u0417\u0430\u043f\u0440\u043e\u0441\u043e\u0432
```
Comment (LonwoLonwo) [inline plugins/org.jkiss.dbeaver.core/OSGI-INF/l10n/bundle_ru.properties:128]:
```suggestion
command.org.eclipse.ui.views.showView.queryManager=\u041c\u0435\u043d\u0435\u0434\u0436\u0435\u0440 \u0437\u0430\u043f\u0440\u043e\u0441\u043e\u0432
```

**Comment by LonwoLonwo** [comment]
Hello @volodymyrchernyshev 

Sorry for the late response.

Ukrainian language support was added here: https://github.com/dbeaver/dbeaver/pull/21768.

So, please remove the second (_ua) file.

And about the first one (Russian).

I marked all typos that I found. Usually, do not use uppercase letters in nouns in non-English localizations. I marked these places, too.

keyword.org.jkiss.dbeaver.core.keyword.pref.db.general.label - please return it as it was. This is a bunch of keywords — no need to translate them. 
Same for:
keyword.org.jkiss.dbeaver.core.keyword.pref.db.dataformat.label
keyword.org.jkiss.dbeaver.core.keyword.pref.db.qm.label
keyword.org.jkiss.dbeaver.core.keyword.pref.editor.binary.label

**Comment by LonwoLonwo** [comment]
Please, also resolve conflicts in your PR.

**Comment by E1izabeth** [comment]
@volodymyrchernyshev hi, would you solve merge conflict and apply suggestions after review by @LonwoLonwo ?
Currently this PR is stale.

**Comment by LonwoLonwo** [comment]
Unfortunately, we need to close this PR. We can't merge it without the required changes.

**Pull request closed without merging**